examples/pytorch/audio-classification/prepare_commonlanguage.py

```diff
@@ -0,0 +1,66 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import glob
+import json
+import os
+import tarfile
+
+from datasets.utils.file_utils import http_get
+
+from transformers import logging
+
+
+logging.set_verbosity_info()
+logger = logging.get_logger(__name__)
+
+DATASET_URL = "https://zenodo.org/record/5036977/files/CommonLanguage.tar.gz?download=1"
+
+
+def prepare_commonlanguage(download_directory):
```

Comment:
Can't we add the common language to `datasets` https://github.com/huggingface/datasets?

Reply:
Also this script doesn't seem to be used - why is it here exactly?